### PR TITLE
Make reg order consistent across x86-*

### PR DESCRIPTION
**Pull request by SchumBlubBlub**

### CODE REVIEW

File: libr/debug/p/native/linux/reg/linux-x86.h [hunk 1]
```diff
@@ -26,14 +24,16 @@ return strdup (
 "gpr	dx	.16	8	0\n"
 "gpr	dh	.8	9	0\n"
 "gpr	dl	.8	8	0\n"
+"gpr  esi .32 12  0\n"
```
Comment (radare) [inline libr/debug/p/native/linux/reg/linux-x86.h]: indent columns with tabs

File: libr/debug/p/native/linux/reg/linux-x86.h [hunk 2]
```diff
@@ -26,14 +24,16 @@ return strdup (
...
+"gpr  di  .16 16  0\n"
 "gpr	esp	.32	60	0\n"
 "gpr	sp	.16	60	0\n"
 "gpr	ebp	.32	20	0\n"
 "gpr	bp	.16	20	0\n"
-"gpr	esi	.32	12	0\n"
-"gpr	si	.16	12	0\n"
-"gpr	edi	.32	16	0\n"
-"gpr	di	.16	16	0\n"
+"gpr  eip .32 48  0\n"
```
Comment (radare) [inline libr/debug/p/native/linux/reg/linux-x86.h]: use tabs

**Comment by radare** [comment]
use tabs everywhere

File: libr/debug/p/native/linux/reg/linux-x86.h (hunk 1 above)
Comment (SchumBlubBlub) [inline libr/debug/p/native/linux/reg/linux-x86.h]: fixed.

File: libr/debug/p/native/linux/reg/linux-x86.h (hunk 2 above)
Comment (SchumBlubBlub) [inline libr/debug/p/native/linux/reg/linux-x86.h]: fixed.

**Comment by SchumBlubBlub** [comment]
Done.

**Pull request merged**